requirements.txt

```diff
@@ -0,0 +1,6 @@
+black
+flake8
+matplotlib
+mypy
+numpy
+pytest
```

Comment:
Maybe move these dependencies to `requirements.in` and then using [pip-tools](https://pypi.org/project/pip-tools/) generate `requirements.txt` to get reproducible build?

Reply:
Not needed. As the tools get better, we get better tests.